db/migrate/20181119145146_remove_paperclip_column.rb

```diff
@@ -0,0 +1,8 @@
+class RemovePaperclipColumn < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :users, :avatar_file_name if column_exists? :users, :avatar_file_name
+    remove_column :users, :avatar_content_type if column_exists? :users, :avatar_content_type
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Rails/ReversibleMigration: remove_column(without type) is not reversible.